src/entry_point.py

```diff
@@ -201,6 +256,311 @@ def _constructor_menuinst(prefix, pkg_names=None, root_prefix=None, remove=False
         install(str(json_path), remove=remove, prefix=prefix, root_prefix=root_prefix)
 
 
+def _is_subdir(directory: Path, root: Path) -> bool:
```

Comment:
Initially I thought this would be something about `CONDA_SUBDIR`, but it's not, right?

Reply:
Correct, it does not refer to `CONDA_SUBDIR`, but to subdirectories on the file system.

I created this helper function because I need it in a few places during the uninstallation process.